server/server/src/static_files.rs

```diff
@@ -215,7 +155,7 @@ async fn delete_sync_file(
 
 #[post("/{table_name}/{record_id}")]
 async fn upload_sync_file(
```

Comment:
I've mentioned in PR description that this is now split into two methods, easier to avoid leaking error and logging